aws/extensions/computetask_linux_volumemount/extension.ftl

```diff
@@ -75,9 +78,7 @@
         [#local deviceId = ""]
         [#local osMount = ""]
 
-        [#if (volume.Enabled)!true
-                && ((volume.MountPath)!"")?has_content
-                && ((volume.Device)!"")?has_content ]
+        [#if volume.Enabled && volume.MountPath?? && volume.Device?? ]
```

Comment:
semantics are slightly different. This checks that the attribute exists but could be empty. The previous code checked for an actual value.

Is that intentional?

Reply:
Yeah this was intentional. The MountPath is an optional value to allow for creating a disk but not formatting it. 
Providing MountPath "" as a default felt weird for that action. 

Is that mount to the root or is that don't mount? leaving it as doesn't exist felt better